selfdrive/car/toyota/values.py

```diff
@@ -1632,5 +1632,8 @@ class CAR:
 
 NO_DSU_CAR = TSS2_CAR | set([CAR.CHR, CAR.CHRH, CAR.CAMRY, CAR.CAMRYH])
 
+# these cars have full speed ACC, but require a resume button or gas pedal press to resume from a standstill
+NO_AUTO_RESUME_CAR = set([CAR.PRIUS, CAR.LEXUS_CTH, CAR.LEXUS_NX, CAR.LEXUS_NXH, CAR.LEXUS_RX, CAR.LEXUS_RXH])
```

Comment:
I'm only 100% confident about the TSS-P Prius since I've been running this code for a while now. The other cars in this list are extrapolated from `stop_and_go = True` in `interface.py` and not `NO_STOP_TIMER_CAR` in `values.py`.

Reply:
let's just stick to the prius for now then